### ISSUE TITLE
No support for other resolutions

### ISSUE THREAD

**Issue by xesdiny** [THUDM/CogVideo#180]
### Feature request  / 功能建议

On Relase [Cogvideox-5B](https://huggingface.co/THUDM/CogVideoX-5b),Video Resolution Only 480P.

### Motivation / 动机

I think Like Open-Sora by colossalai can be achieved using multi-bucket + 3D RoPE. However, each device can only use the DDP single-instance training mode to synchronize and communicate its own gradients after backpropagation. I am not sure whether I can still use the context parallellism module.
But I have tried Cogvideox-2b to make it feasible on a single GPU, and Tensor parallellism + context parallellism needs to be modified based on colossalai Kernel.

### Your contribution / 您的贡献

None

**Comment by zRzRzRzRzRzRzR** [comment]
We will try to launch a fine-tuning solution on the diffusers version. We will pay attention to the implementation and optimization of context parallelism. Due to limited human resources, the exploration in this area is progressing relatively slowly, but fine-tuning is a high-priority task for us at the moment
